### PR TITLE
WIP: Enable graying of create DOI when zenodo credentials are not linked

### PR DESCRIPTION
**Pull request by wshands**
addresses https://ucsc-cgl.atlassian.net/browse/SEAB-342

Need feedback on implementation...

### CODE REVIEW

**Comment by garyluu** [comment]
Rebase after I merge in my fix

File: src/app/workflow/workflow.component.ts [hunk 1]
```diff
@@ -238,6 +245,24 @@ export class WorkflowComponent extends Entry implements AfterViewInit {
     }
   }
 
+  // Show linked services in the UI
+  private setAvailableTokens(tokens) {
+      const found = tokens.find(token => token.tokenSource === 'zenodo.org');
```
Comment (coverbeck) [inline src/app/workflow/workflow.component.ts]: `this.zenodoAccountIsLinked = tokens.some(token => token.tokenSource === 'zenodo.org');`

File: src/app/workflow/workflow.component.ts [hunk 2]
```diff
@@ -253,6 +278,9 @@ export class WorkflowComponent extends Entry implements AfterViewInit {
       }
       this.setUpWorkflow(workflow);
     });
+    this.tokenQuery.tokens$.subscribe((tokens: Token[]) => {
+      this.setTokens(tokens);
```
Comment (coverbeck) [inline src/app/workflow/workflow.component.ts]: I don't think you need to save the tokens; just set zenodoAccountIsLinked here.

**Comment by garyluu** [comment]
Could use a simple test for this

cy.get('[data-cy=thing]').should(be.disabled)

as I'm not entirely sure how this works

File: src/app/workflow/workflow.component.ts (hunk 1 above)
Comment (wshands) [reply src/app/workflow/workflow.component.ts]: done

File: src/app/workflow/workflow.component.ts (hunk 2 above)
Comment (wshands) [reply src/app/workflow/workflow.component.ts]: done

**Comment by wshands** [comment]
Closing so I can create a non draft pull request

**Pull request closed without merging**